install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -11,144 +11,406 @@
-== SupplementalGroups
-
-Supplemental groups are regular Linux groups. When a process runs in Linux, it
-has a UID and one or more GIDs. Usually this is just the UID and GIDs of the
-user who started the process, but in the case of containers you have the option
-of setting these attributes for the container's main process.
-
-The `*SupplementalGroups*` option of the pod `*SecurityContext*` stanza enables
-you to specify a list of groups which will be added to the main process of every
-container in the pod. This is useful because it enables you to give your pods
-group based access to volumes.
-
+(*Note:* "project" and "namespace" are used interchangeably throughout this document)
+
+This topic provides a general guide on pod security as it relates to volume security.
+For information on pod-level security in general, please see
+link:../../admin_guide/manage_scc.html[Managing Security Context Constraints (SCC)]
+and
+link:../../architecture/additional_concepts/authorization.html#security-context-constraints[SCC concepts].
+
+As a quick background, the Openshift
+link:../../architecture/additional_concepts/storage.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage and gives users
+a way to request those resources without requiring knowledge of the underlying infrastructure.
+Persistent volumes (PV) are global resources and not tied to a specific project (namespace). 
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
+volume claims (PVC)] are local to the project, and multiple PVCs within the same project
+can bind to the same PV. However, once a PVC binds to a PV, that PV cannot be bound by a claim
+outside of the first claim's namespace. If the underlying storage needs to be accessed by
+multiple projects then each project needs its own PV, which can point to the same physical
+storage. In this sense, a bound PV is tied to a namespace. For a detailed PV and PVC example,
+see the guide for
+https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and MySQL using NFS].
+
+*NOTE:*
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+
+Accessing persistent storage requires coordination between the cluster (and/or storage)
+administrator and the end-developer. The cluster admin creates persistent volumes (PVs),
+which abstract the underlying physical storage. The developer creates pods and,
+optionally, persistent volume claims (PVCs), which bind to PVs, based on matching
+criteria, such as capacity. For the admin, granting pods access to PVs involves knowing
+the group ids and/or user id assigned to the actual storage, selinux considerations, and,
+finally, ensuring that these ids are allowed in the range of legal ids defined for the
+project and/or the SCC that matches the requirements of the pod.
+
+Group ids, the user id, and selinux values are defined in the `*SecurityContext*` stanza
```

Comment:
@adellape Do you typically use the names of fields as they are in the golang type definitions, or as they surface to the user in API yaml/json?  If (2), this should be `selinuxOptions`